components/personCard/personCard.tsx

```diff
@@ -46,7 +47,11 @@ export function PersonCard({ person }: IPersonProps) {
     <div ref={card} className={styles.card} style={{ borderBottomColor: theme }}>
       <div
         className={styles.card__img}
-        style={{ backgroundImage: `url(https://github.com/${github}.png)` }}
+        style={{
+          backgroundImage: customImage
+            ? `url(images/people/${github}.jpeg)`
```

Comment:
@EladTal14 
if you added new prop for this `customImage` why you not use it?

other then that maybe it will be more clean to create dedicate function for this behavior compare a terny statement

Reply:
the new prop is a truth/false variable that tells the program if the user wants to use a custom image or the image from github

Reply:
understood